src/Analysis/Engine/Test/LanguageServerTests.cs

```diff
@@ -738,14 +738,14 @@ class D:
 
             // f
             var expected = new[] {
-                "Definition;(2, 5) - (2, 6)",
-                "Value;(2, 1) - (3, 11)",
-                "Reference;(5, 1) - (5, 2)",
-                "Reference;(10, 1) - (10, 2)",
-                "Reference;(17, 6) - (17, 7)"
+                "Definition;(1, 4) - (1, 5)",
```

Comment:
Should we move this one to the references test class as well?

Reply:
Probably... I guess it is not worth keeping LS tests separate since it is all about LS.

Reply:
Ok.